spk/transmission/Makefile

```diff
@@ -1,9 +1,12 @@
 SPK_NAME = transmission
 SPK_VERS = 4.0.4
-SPK_REV = 23
+SPK_REV = 24
 SPK_ICON = src/transmission.png
 
-DEPENDS = cross/$(SPK_NAME)
+DEPENDS = cross/transmission
+
+REQUIRED_MIN_DSM = 7.0
```

Comment:
This may have already been answered but curious as to why we need this duplicated here if it's already in the `cross` Makefile.

Reply:
1. for clarity (I don't want to look into cross packages for such dependencies, when I have the focus on the spk files)
2. to avoid any download for unsupported archs/DSM version (I saw such downloads but can't remember the related packages, probably this is still an issue with BUILD_DEPENDS).

so we need the definition in cross and spk (as we might build in cross too).